tests/integration/api/test_rulebook.py

```diff
@@ -426,7 +454,7 @@ def test_list_actions_from_audit_rule_ordering(
     ordering_field,
 ):
     response = admin_client.get(
-        f"{api_url_v1}/audit-rules/{audit_rule_2.id}/actions/?ordering="
+        f"{api_url_v1}/audit-rules/{audit_rule_2.id}/actions/?order="
```

Comment:
@Dostonbek1 Will this query difference have to be coordinated with UI @lgalis? Is the EDA UI using ordering?